rules/opentitan.bzl

```diff
@@ -86,7 +86,7 @@ def create_prod_key(name, label):
         CONST.LCV.RMA,
     ])
 
-def create_keyset(rsa_key, spx_key):
+def create_key_duplet(rsa_key, spx_key):
```

Comment:
This is purely personal preference, but I don't like the term "duplet" and we'll have to change it if ever becomes more than two things.

I think the word "tuple" or "struct" would fit a bit better.
- Tuple may have some implied meaning given starlark's pythonic legacy (ie: an immutable sequence).
- The entity that is created is actually a struct.

Reply:
same here :) I didn't want to use `pair` or `tuple` for the same reasons. I'm ok with `struct`. I'll update and merge, thanks!